application-collabora-ui/src/main/resources/Collabora/Code/Main.xml

```diff
@@ -219,6 +266,26 @@
     }).fail(function(jqxhr, textStatus, error) {
       new XWiki.widgets.Notification(errorMessage + error, 'error');
     });
+
+    // Listen to messages send by Collabora server.
```

Comment:
Again, I don't think these messages are send by the Collabora **sever** but rather by Collabora **editor** (i.e. the JavaScript code loaded in the iframe).